docs/source/data-format-5to6.ipynb

```diff
@@ -14,31 +14,37 @@
     "\n",
-    "| Type       | v0.5.x          | v0.6.0                 | Rationale       |\n",
-    "| :--------- | :-------------- | :--------------------- | :-------------- |\n",
+    "| Type       | v0.5.x          | v0.6.0                 | Rationale and notes      |\n",
+    "| :--------- | :-------------- | :--------------------- | :----------------------- |\n",
     "| Group      | `Beam`          | `Sonar/Beam_group1`    | Convention compliance |\n",
     "| Group      | `Beam_power`    | `Sonar/Beam_group2`    | Convention compliance |\n",
-    "| Coordinate | `frequency`     | `channel`              | Avoid problems with channels with duplicated frequencies |\n",
+    "| Group      | `Vendor`        | `Vendor_specific`      | Convention compliance |\n",
+    "| Coordinate | `frequency`     | `channel`              | Accommodate channels with duplicated frequencies. The new variable `frequency_nominal` was introduced |\n",
     "| Coordinate | `range_bin`     | `range_sample`         | Better intuitive understanding of data |\n",
-    "| Coordinate | `quadrant`      | `beam`                 | Convention compliance |\n",
-    "| Coordinate | `location_time` | `time1`                | Convention compliance |\n",
-    "| Coordinate | `mru_time`      | `time2`                | Convention compliance |\n",
-    "| Variable   | `heave`         | `vertical_offset`      | Convention compliance |\n",
-    "| Variable   | `src_filenames` | `source_filenames`     | Convention compliance |\n",
+    "| Coordinate | `quadrant`      | `beam`                 | Convention compliance. This `Beam_groupX` coordinate was added when it did not exist. |\n",
+    "| Coordinate | `location_time` | `time1`                | Convention compliance. In `Platform` group |\n",
+    "| Coordinate | `mru_time`      | `time2`                | Convention compliance. In `Platform` group |\n",
+    "| Variable   | `heave`         | `vertical_offset`      | Convention compliance. In `Platform` group |\n",
+    "| Variable   | `src_filenames` | `source_filenames`     | Convention compliance. In `Provenance` group. Also converted from global attribute to variable |\n",
     "\n",
-    "The PRs and discussions related to these changes can be found in the [Release notes](PUT LINK)."
+    "Other changes included:\n",
+    "- Adding previously missing, mandatory convention variables. When no data are available to populate them, these are filled with null (`NaN`) values.\n",
+    "- Moving variables from one group to another, particularly from the Beam groups to `Platform` and `Vendor`. These variables were not typically not part of the convention.\n",
+    "- The Beam_groupX `beamwidth_receive_athwartship` and `beamwidth_transmit_athwartship` variables were consolidated into `beamwidth_twoway_athwartship`. Likewise for `beamwidth_receive_alongship` and `beamwidth_transmit_alongship`.\n",
```

Comment:
```suggestion
    "- The Beam_groupX `beamwidth_receive_athwartship` and `beamwidth_transmit_athwartship` variables were consolidated into `beamwidth_twoway_athwartship` because the EK60 and EK80 echosounders do not store one-way transmit or receive beam widths. Likewise for `beamwidth_receive_alongship` and `beamwidth_transmit_alongship`.\n",
```